silkworm/db/datastore/snapshots/snapshot_repository.cpp

```diff
@@ -157,8 +161,9 @@ void SnapshotRepository::reopen_folder() {
     bundles_ = bundles;
     lock.unlock();
 
-    SILK_INFO << "Total reopened bundles: " << bundles_count()
-              << " max block available: " << max_block_available();
+    SILK_INFO << "Total reopened " << magic_enum::enum_name(kind_) << " bundles: " << bundles_count()
+              << (kind_ == RepositoryKind::blocks ? " max block available: " : " max timestamp available: ")
+              << (kind_ == RepositoryKind::blocks ? max_block_available() : max_timestamp_available());
```

Comment:
both blocks and txns are timestamps in our terminology
max_block_available and max_timestamp_available return exactly the same value
max_block_available() is a legacy method and shouldn't be used within the datastore, I ought to refactor it, but didn't fine time

I'd keep it simple for now and just print `max timestamp` here (a log reader will understand what the unit is based on the repo name)

if we want to make it prettier and identify the unit name (blocks or txns), I'd add a method to the StepToTimestampConverter interface